### ISSUE TITLE
summary of errors

### ISSUE THREAD

**Issue by namkkakkaka** [tesa-klebeband/RTL8720dn-Deauther#22]
1. When scanning for wifi, you will get an error not found! at 192.168.1.1/rescan 2. When deauth is performed, all nearby wifi is affected, not just the wifi in the selected list 3. Sometimes refreshing the page will also result in a not found error! at website 192.168.1.1

**Comment by tesa-klebeband** [comment]
Hello, I already noticed the not found errors and will fix them in the next few days. However, thanks for pointing out the error where all wifis are affected. I myself didn't encounter it, but it might be due to a different browser etc. Could you please post your exact setup, so I can test it myself to fix it.

**Comment by namkkakkaka** [comment]
@tesa-klebeband when I select the wifi network in the scanned wifi list and enter reason 2 in the reason list and press start deauth then the wifi networks that I did not select are also affected such as reduced speed or cannot connect

**Comment by tesa-klebeband** [comment]
Okay, could you post which browser you are using. Might be an issue with creating the post request and processing it.

**Comment by namkkakkaka** [comment]
@tesa-klebeband I use chrome browser on android phone

**Comment by tesa-klebeband** [comment]
Reduced speed is normal, as the code is currently sending many frames. You can change the number and time between frames in the loop() function, and check if that fixes it for you.

**Comment by namkkakkaka** [comment]
@tesa-klebeband So sending too many frames will affect networks that are not in the selected list.

**Comment by tesa-klebeband** [comment]
Yeah kinda, it slows down the other networks speed too, but it's weird that is doesn't let you connect to them either. Also, only the networks on the channel of the to be deauthed network should be affected.

**Comment by namkkakkaka** [comment]
@tesa-klebeband yes the network not selected sometimes can not connect or will show no internet

**Comment by tesa-klebeband** [comment]
Try changing the speed the frames are sent in
```cpp
delay(5);
```
to a higher value like 10.

**Comment by droid4gamers** [comment]
@tesa-klebeband how can I hide the SSID? Thanks!

**Comment by tesa-klebeband** [comment]
Hi, I believe you can hide it in an argument to the WiFi.apbegin() function. I'm not sure which one, it should be written in the docs, or in the function definition. If you need any more help, feel free to ask.

**Comment by droid4gamers** [comment]
Hello @tesa-klebeband thank you for replying. By any chance do you have a source website where I can see what command to put on how to hide my SSID? Sorry if you find my question lame. I don't have any background on coding or programming. Just searching on Google and YouTube. I'm really noob on this. Thanks again for your response. Btw I appreciate you sharing this.

**Comment by droid4gamers** [comment]
> Hi, I believe you can hide it in an argument to the WiFi.apbegin() function. I'm not sure which one, it should be written in the docs, or in the function definition. If you need any more help, feel free to ask.

Thank you @tesa-klebeband. It's all good now.

**Comment by uacode** [comment]
@tesa-klebeband Hello in html you should send the network as array (for multiple usage)
And correctly process it in a loop (how I don't know exactly, I'm not writing for Arduino)

`response += "<td><input type='checkbox' name='network[]' value='" + String(i) + "'></td>";`
